graphistry/compute/predicates/str.py

```diff
@@ -14,8 +14,24 @@ def __init__(self, pat: str, case: bool = True, flags: int = 0, na: Optional[boo
         self.regex = regex
 
     def __call__(self, s: SeriesT) -> SeriesT:
-        return s.str.contains(self.pat, self.case, self.flags, self.na, self.regex)
-    
+        is_cudf = hasattr(s, '__module__') and 'cudf' in s.__module__
+
+        # workaround cuDF not supporting 'case' and 'na' parameters https://docs.rapids.ai/api/cudf/stable/user_guide/api_docs/api/cudf.core.accessors.string.stringmethods.contains/
+        if is_cudf:
```

Comment:
Can we have same universal behavior ? Afaict that means just removing the outer if here